architecture-decision-records/0035-metadata-csv-utf-8-file-validation.md

```diff
@@ -0,0 +1,30 @@
+# 35. Metadata CSV file UTF-8 Validation
+
+**Date**: 2024-10-07
+
+## Context
+
+Digital assets sent to The National Archives (TNA) are transferred with associated metadata in CSV format. The entry point for digital assets to TNA will be through the services provided by the Digital Selection and Transfer team.
+
+The structure and integrity of the CSV file is checked before the actual [data is validated](0034-metadata-validation.md) 
+
+## Structure and integrity checks
+
+1. **Virus check**: All files uploaded to TDR are checked for viruses using the virus check lambda
```

Comment:
Don't need to mention the virus check as this is only about the UTF-8 validation